homeassistant/components/imeon_inverter/strings.json

```
@@ -214,6 +214,17 @@
       "energy_battery_consumed": {
         "name": "Today battery-consumed energy"
       }
+    },
+    "select": {
+      "manager_inverter_mode": {
+        "name": "Inverter mode",
+        "state": {
+          "smart_grid": "Smart-grid",
```

"Smart grid" does not use a hyphen while "on-grid" and "off-grid" are correct:

```suggestion
          "smart_grid": "Smart grid",
```